README.md

```diff
@@ -1,2 +1,201 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<<<<<<< HEAD
+<<<<<<< desktop-version-completed
+> Add a link to your deployed project.
+
+- [Live Demo Link]https://samtush.github.io/Startup-battlefield/
+<<<<<<< desktop-version-completed
+=======
+=======
+- [Live Demo Link](https://samtush.github.io/Startup-battlefield/)
+>>>>>>> main
+>>>>>>> main
+=======
+- [Live Demo Link](https://samtush.github.io/Startup-battlefield/)
+>>>>>>> main
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LOOM VIDEO -->
+
+## 🚀 Loom video <a name="loom-video"></a>
+
+- [Live Demo Link](https://www.loom.com/share/4a704710d1964b7390cc2a2003575079)
+<<<<<<< Loom video of the project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Clone the repo and contribate to the site.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Internet connection to clone the project
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/SamTush/Startup-battlefield.git
+```
+
+### Install
+
+Install this project with: VS code or any other code editor.
+
+### Usage
+
+To run the project: Go live in VS or get a live server for your code editor.
+
+### Run tests
+
+To run tests, run linter:
+
+### Deployment
+
+You can deploy this project using: github
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> All collaborators of this project.
+
+👤 **Samson**
+
+- GitHub: [@githubhandle](https://github.com/SamTush)
+- Twitter: [@twitterhandle](https://twitter.com/samo44o)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/samson-tush-4a7386216/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Turn Behance design to a project.
+Add other pages in the future.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, feel free to comment and contribute to it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thanks to anyone viewing this project and giving a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[What is this website?]**
+
+  - [A design by Cindy Shin in Behance]
```

Comment:
- [ ] Please move the credit you are giving to the author of the original design to the `Acknowledgements` section because that is the proper place for it.👌